### PR TITLE
Pr/2102

### PR DESCRIPTION
**Pull request by wickedknock**
- updated super linter for python and golang
- added lint.js to test linters locally

### CODE REVIEW

**Review by jbampton — CHANGES_REQUESTED** [review]
Please remove `lint.js` as we use pre-commit locally on git commit

File: .gitignore [hunk 1]
```diff
@@ -27,3 +27,4 @@ workers.dev/preston/node_modules/
 workers.dev/preston/worker/
 .vscode/settings.json
 .vscode/c_cpp_properties.json
+.env
```
Comment (jbampton) [inline .gitignore:30]:
```suggestion
.env

```

File: .eslintrc.json [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+        "error",
+        "single"
+      ],
+      "semi": [
+        "error",
+        "always"
+      ]
+    }
+  }
+
```
Comment (jbampton) [inline .eslintrc.json:33]:
```suggestion
 
```

**Review by jbampton — COMMENTED** [review]
Keep lint.js change my mind !!

**Comment by BaseMax** [comment]
Thanks.

**Comment by wickedknock** [comment]
@jbampton  Its only failing on dead links now , should I start working on other issues?

**Comment by jbampton** [comment]
https://app.codacy.com/gh/slurpcode/slurp/pullRequest?prid=11440795

**Review by jbampton — CHANGES_REQUESTED** [review]
https://deepsource.io/gh/slurpcode/slurp/run/99c356b4-0ac6-4341-accb-31462fc94470/javascript/

**Review by jbampton — APPROVED** [review]

**Pull request merged**